Console.CodingTracker.mefdev/Controller/CodingController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using CodingLogger.Models;
+using Spectre.Console;
+using Spectre.Console.Cli;
+
+namespace CodingLogger.Controller
```

Comment:
Remember [code-conventions](https://thecsharpacademy.com/article/58/code-conventions)

Use the newest namespaces.
From C# 10 we are able to ditch the namespace block and save one indentation level, which optimizes the use of space.

![code-convention-6](https://github.com/user-attachments/assets/5c71f2a5-4348-41e4-a1c1-5a6751890c29)